content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -27,7 +27,7 @@ Kubernetes as a project currently supports and maintains [GCE](https://git.k8s.i
   controller with [community](https://www.getambassador.io/docs) or 
   [commercial](https://www.getambassador.io/pro/) support from [Datawire](https://www.datawire.io/).
 * [AppsCode Inc.](https://appscode.com) offers support and maintenance for the most widely used [HAProxy](https://www.haproxy.org/) based ingress controller [Voyager](https://appscode.com/products/voyager). 
-* [AWS ALB Ingress Controller](https://github.com/kubernetes-sigs/aws-alb-ingress-controller) enables ingress using the [AWS Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/).
+* [AWS Load Balancer Controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller) (formerly known as AWS ALB Ingress Controller) enables ingress using the [AWS Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/).
```

Comment:
```suggestion
* [AWS Load Balancer Controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller) (formerly known as AWS ALB Ingress Controller) enables ingress using the [AWS Elastic Load Balancing](https://aws.amazon.com/elasticloadbalancing/).
```

Reply:
Hi @chlunde . Thanks for opening this pull request.
The changes look good.
What do you think about incorporating this suggestion?